src/components/SystemScoreSection.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <div class="lg:hidden flex px-4 py-2">
+        <h2 class="uppercase font-bold text-xl">Your System:</h2>
+        <button class=" ml-auto" @click="editSelections()">
+            <i class="pi pi-cog"
+                v-tooltip="'Click here to edit your system selections and recalibrate the top ten list'"></i>
+        </button>
+    </div>
+    <div>
+        <p class="system-score-row px-4 w-max max-w-full mx-auto my-auto uppercase font-bold text-ctid-black">
+            <span v-for="monitoringType of Object.keys(this.calculatorStore.systemScore)" :key="monitoringType"
+                class=" mr-4 "><span class="highlight">{{
+            getScoreText(monitoringType) }}</span>
+                {{ monitoringType }} Monitoring </span>
+            <span v-for="filter of Object.keys(this.calculatorStore.activeFilters)" :key="filter">
+                <span class="mr-4" v-if="this.calculatorStore.activeFilters[filter].length > 0">
```

Comment:
Depending on viewport width, this span can have a linebreak in the middle. Can you add a CSS style like `white-space: nowrap;` to prevent linebreaks inside this span?

And do the same thing for the other spans in this header (monitoring and edit selection).